src/components/TeacherComponents/OnboardingForm/OnboardingForm.test.tsx

```diff
@@ -102,9 +104,23 @@ describe("Onboarding form", () => {
       mockRouter.setCurrentUrl("/onboarding?returnTo=/downloads");
     });
 
-    it("onboards the user through Clerk", async () => {
-      jest.spyOn(onboardingActions, "onboardUser");
+    afterEach(() => {
+      jest.spyOn(onboardingActions, "onboardUser").mockReset();
+    });
+
+    it("only allows the form to be submitted once", async () => {
+      renderForm(formState);
+
+      const continueButton = screen.getByText("Continue");
+      await userEvent.click(continueButton);
+      await userEvent.click(continueButton, {
+        pointerEventsCheck: PointerEventsCheckLevel.EachTarget,
```

Comment:
The second click will fails because pointer events are disabled on the immediate child of the disabled button. We still want to perform the click to verify that nothing happens (I.e the button is disabled and the form cannot be submitted — it returns early).